swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2023-09-04/redhatopenshift.json

```diff
@@ -2224,6 +2228,14 @@
         "modelAsString": true
       }
     },
+    "PreconfiguredNSG": {
```

Comment:
I think we want to use the x-ms-enum value here as well, similar to OUtboundType.  I can't remember why this is the way it is, but i think there's a good reason.

Reply:
Agreed.  Made the required changes and regenerated to have `x-ms-enum` included.